README.md

```diff
@@ -44,8 +44,21 @@ $ brew install dnsmasq
 $ sudo brew services start dnsmasq
 ```
 
+### What services might you have available? Let's find out! ###
```

Comment:
The difference of tone stands out from other examples, which are all written in a more imperative mood.

The text from `brew services --help` is "``List all services managed by `brew services` ``"